### PR TITLE
Add release notes for v.1.13.0

### PR DESCRIPTION
**Pull request by alexandra-simeonova**
**Description**

Changes proposed in this pull request:

- Add release notes for Luigi v.1.13.0 to introduce the new changes in our blog 

**Related issue(s)**

### CODE REVIEW

File: blog/2021-06-07-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Luigi v1.13.0
+seoMetaDescription: Release notes for Luigi v1.13.0
+author:
+  - Aleksandra Simeonova
+layout: blog
+---
+
+Luigi 1.13.0 is the latest Luigi release. It includes updates of Fundamental Styles, new options in the user settings, as well as other features and bugfixes. Read the release notes below to learn more.
```
Comment (UlianaMunich) [inline blog/2021-06-07-release-notes.md:9]:
```suggestion
Luigi 1.13.0 is the latest Luigi release. It includes updates of Fundamental Library Styles, new options in the user settings, as well as other features and bug fixes. Read the release notes below to learn more.
```

File: blog/2021-06-07-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+author:
+  - Aleksandra Simeonova
+layout: blog
+---
+
+Luigi 1.13.0 is the latest Luigi release. It includes updates of Fundamental Library Styles, new options in the user settings, as well as other features and bug fixes. Read the release notes below to learn more.
+
+<!-- Excerpt -->
+
+#### Breaking changes in Fundamental Styles
```
Comment (UlianaMunich) [inline blog/2021-06-07-release-notes.md:13]:
```suggestion
#### Breaking changes in Fundamental Library Styles
```

File: blog/2021-06-07-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+<!-- Excerpt -->
+
+#### Breaking changes in Fundamental Library Styles
+
+With Luigi v1.13.0, the new v0.17.0 of Fundamental Library Styles was included. It led to having breaking changes in the Luigi Dialog/Modal layout and Shellbar Counter. New HTML attributes were added to those components to increase accessibility. You can see the new layout of these components here: [Dialog](https://sap.github.io/fundamental-styles/?path=/docs/components-dialog--default-dialog) and [Shellbar Counter](https://sap.github.io/fundamental-styles/?path=/docs/components-shellbar--primary).
+
+#### Extend options of enum type in user settings
+
+In Luigi's [user settings](https://docs.luigi-project.io/docs/user-settings), an `enum` array can now can store options objects. You can use them if you want to add more options in the user settings for the user to choose from. These objects need `value` and `label` as key. For example:
```
Comment (UlianaMunich) [inline blog/2021-06-07-release-notes.md:19]:
```suggestion
In Luigi's [user settings](https://docs.luigi-project.io/docs/user-settings), an `enum` array can now store options objects. You can use them if you want to add more options in the user settings for the user to choose from. These objects need `value` and `label` as key. For example:
```

**Review by UlianaMunich — APPROVED** [review]

File: blog/2021-06-07-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+layout: blog
+---
+
+Luigi 1.13.0 is the latest Luigi release. It includes updates of Fundamental Library Styles, new options in the user settings, as well as other features and bug fixes. Read the release notes below to learn more.
+
+<!-- Excerpt -->
+
+#### Breaking changes in Fundamental Library Styles
+
+With Luigi v1.13.0, the new v0.17.0 of Fundamental Library Styles was included. It led to having breaking changes in the Luigi Dialog/Modal layout and Shellbar Counter. New HTML attributes were added to those components to increase accessibility. You can see the new layout of these components here: [Dialog](https://sap.github.io/fundamental-styles/?path=/docs/components-dialog--default-dialog) and [Shellbar Counter](https://sap.github.io/fundamental-styles/?path=/docs/components-shellbar--primary).
```
Comment (hardl) [inline blog/2021-06-07-release-notes.md:15]: Maybe we could add sth. like
"As always this does only affect you if you have customized fundamentals css in your app" or sth. like that

**Review by hardl — CHANGES_REQUESTED** [review]
See my comment, rest looks good 👍

**Review by hardl — APPROVED** [review]

**Pull request merged**